### PR TITLE
Feature-routes-and-navigation

### PR DESCRIPTION
**Pull request by rohxnn**

### CODE REVIEW

File: src/app/login-form/login-authentication-guard/loginAuthentication.guard.ts
Comment (jobz123) [inline src/app/login-form/login-authentication-guard/loginAuthentication.guard.ts:1]: Folder name and file name should be same!!!!

File: src/app/login-form/login-form.model.ts
Comment (jobz123) [inline src/app/login-form/login-form.model.ts:1]: Login form should be in a dedicated folder, as discussed previously, but has not been implemented yet!!!!

File: src/app/login-form/service/login.service.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if (this.user.email === email && this.user.password === password) {
+      this.setToken('abcdefg');
+      this.route.navigate(['/admin/home']);
+    } else {
+      alert('Wrong email or password');
+    }
+  }
+
+  logout() {
+    this.setToken('');
```
Comment (jobz123) [inline src/app/login-form/service/login.service.ts:36]: Token should be cleared instead of updating the token. Use clear()

**Review by jobz123 — CHANGES_REQUESTED** [review]
Added some uupdations.

**Review by jobz123 — CHANGES_REQUESTED** [review]
change the commit message

File: src/app/login-form/service/login.service.ts (hunk 1 above)
Comment (rohxnn) [reply src/app/login-form/service/login.service.ts]: All changes are done!

File: src/app/admin/components/customer-data/customer-data.component.html
Comment (jobz123) [inline src/app/admin/components/customer-data/customer-data.component.html:1]: change the folder and file name customer-detail

File: src/app/admin/components/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <th scope="col">PLANS</th>
+        <th scope="col">PH_NUM</th>
+        <th scope="col">FIRST_NAME</th>
+        <th scope="col">LAST_NAME</th>
+        <th>ACTION</th>
+      </tr>
+    </thead>
+    <tbody>
+      <ng-container *ngIf="customerDetails?.length!==null;else noData">
+        <tr *ngFor="let customer of customerDetails; let i=index">
```
Comment (jobz123) [inline src/app/admin/components/home/home.component.html:19]: Need to navigate to detail page while clicking any data/row

**Review by jobz123 — CHANGES_REQUESTED** [review]
Changes added

File: src/app/admin/components/admin/admin.component.html
Comment (jobz123) [inline src/app/admin/components/admin/admin.component.html:1]: change the folder and file name to admin-dashboard

**Review by jobz123 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**